agent/pagebroker/daemon.cpp

```
@@ -29,6 +42,89 @@ Stop(int)
+  const int flags = fcntl(listener, F_GETFL);
+  if (flags < 0 || fcntl(listener, F_SETFL, flags | O_NONBLOCK) < 0)
+    return {errno, std::generic_category()};
+  return {};
+}
+
+std::pair<FileDescriptor, std::error_code>
+CreateListener(const fs::path& socket_path)
+{
+  std::error_code error;
+  fs::remove(socket_path, error);
+  if (error)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)